cron/internal/data/iterator.go

```diff
@@ -64,3 +64,30 @@ func (reader *csvIterator) Next() (RepoFormat, error) {
 	}
 	return reader.next, nil
 }
+
+func MakeNestedIterator(iterators []Iterator) (Iterator, error) {
+	return &nestedIterator{iterators: iterators}, nil
+}
+
+type nestedIterator struct {
+	iterators []Iterator
+	current   int
+}
+
+func (i *nestedIterator) HasNext() bool {
+	for i.current < len(i.iterators) {
+		if i.iterators[i.current].HasNext() {
+			return true
+		}
+		i.current++
+	}
+	return false
+}
+
+func (i *nestedIterator) Next() (RepoFormat, error) {
+	r, err := i.iterators[i.current].Next()
```

Comment:
Replace with `return i.iterators[i.current].Next()` ?

Reply:
I originally had that, but the `wrapcheck` linter complained.
> error returned from interface method should be wrapped: sig: func